tests/data_context/test_base_data_context.py

```diff
@@ -48,17 +56,18 @@ def basic_in_memory_data_context_config_just_stores():
 @pytest.fixture()
 def basic_in_memory_data_context_just_stores(
     basic_in_memory_data_context_config_just_stores,
-):
+) -> AbstractDataContext:
```

Comment:
You added types too!

Reply:
Only at first.... I got lazier as I went 😆